cmd/service.go

```diff
@@ -102,7 +102,12 @@ func (m *ServerMux) handleServiceSignals() error {
 			}
 			runExitFn(nil)
 		case serviceStop:
-			log.Println("Gracefully stopping... (press Ctrl+C again to force)")
+			// Extra space will prevent showing "^CExiting" and rather show "^C Exiting" instead.
+			log.Println(" Exiting")
```

Comment:
Separate signal handlers is not that difficult, but perhaps we can settle for a more generic message like, `Received exit signal (Ctrl+C/SIGTERM/etc)! Attempting to gracefully exit.`